src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier color = new ColorSupplier();
+        ball.setNumber(new Random().nextInt(100));
+        ball.setColor(color.getRandomColor());
+        return ball.toString();
+    }
```

Comment:
Don't use static methods in your solution. Instead of making `getRandomBall` static, create an instance of the Lottery class in your main method and call `getRandomBall` on that instance.